sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/DefaultAzureMonitorOptions.cs

```diff
@@ -31,6 +31,7 @@ public void Configure(AzureMonitorOptions options)
 
             try
             {
+                // TODO: should we replace this with IConfiguration?
```

Comment:
Better, we read it from IConfiguration in distro. Better to address it before GA.

Reply:
lets prioritize this for next week!